prepare/tasks/generation.py

```diff
@@ -3,8 +3,9 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["input", "type_of_input", "type_of_output"],
-        outputs=["output"],
+        inputs={"input": "str", "type_of_input": "str", "type_of_output": "str"},
+        outputs={"output": "str"},
+        prediction_type="Any",
```

Comment:
```suggestion
        prediction_type="str",
```

Reply:
When calling `get_prediction_type` for `normalized_sacrebleu` it returns `Any` as the metrics does not have the `prediction_type` parameter set. So in such a case, we want the attribute to be compatible with a given task?